qemu/tests/win_virtio_driver_install_by_installer.py

```diff
@@ -40,19 +40,24 @@ def run(test, params, env):
     vm = env.get_vm(params["main_vm"])
     session = vm.wait_for_login()
 
-    expected_gagent_version = install_gagent(session, test,
-                                             qemu_ga_pkg,
-                                             gagent_install_cmd,
-                                             gagent_pkg_info_cmd)
-    uninstall_gagent(session, test, gagent_uninstall_cmd)
-    win_uninstall_all_drivers(session, test, params)
+    expected_gagent_version = win_driver_installer_test.install_gagent(
+        session, test, qemu_ga_pkg, gagent_install_cmd, gagent_pkg_info_cmd)
+    win_driver_installer_test.uninstall_gagent(
+            session, test, gagent_uninstall_cmd)
+    win_driver_installer_test.win_uninstall_all_drivers(session,
+                                                        test,
+                                                        params)
     session = vm.reboot(session)
-    install_test_with_screen_on_desktop(vm, session, test, run_install_cmd,
-                                        installer_pkg_check_cmd,
-                                        copy_files_params=params)
-    win_installer_test(session, test, params)
-    check_gagent_version(session, test, gagent_pkg_info_cmd,
-                         expected_gagent_version)
-    driver_check(session, test, params)
+    win_driver_installer_test.install_test_with_screen_on_desktop(
+            vm, session, test, run_install_cmd, installer_pkg_check_cmd,
+            copy_files_params=params)
+    win_driver_installer_test.win_installer_test(session, test, params)
+    win_driver_installer_test.check_gagent_version(
+            session, test, gagent_pkg_info_cmd, expected_gagent_version)
+    win_driver_installer_test.driver_check(session, test, params)
+
+    driver_test_name = params.get("driver_test_name")
+    eval("win_driver_installer_test.%s_test(test, params, vm, **driver_test_params)"
+         % driver_test_name)
```

Comment:
This is too long, also 'driver_test_name' has to exist here.
```suggestion
    test_name = "win_driver_installer_test.%s_test" % params["driver_test_name"]
    eval("%s(test, params, vm, **driver_test_params)" % test_name)
```
Or if we allow it to to skipped:
```suggestion
    test_name = params.get("driver_test_name")
    if test_name:
        test_name = "win_driver_installer_test.%s_test" % test_name
        eval("%s(test, params, vm, **driver_test_params)" % test_name)
```